indico/modules/categories/controllers/management.py

```diff
@@ -117,6 +122,36 @@ def _process(self):
                                                     icon_form=icon_form, logo_form=logo_form)
 
 
+class RHAPIEventMoveRequests(RHManageCategoryBase):
+    def _process_args(self):
+        RHManageCategoryBase._process_args(self)
+        if request.method != 'GET':
+            self.move_requests = parser.parse({
+                'requests': EventRequestList(required=True, category=self.category, validate=not_empty)
+            }, unknown=EXCLUDE)['requests']
```

Comment:
I'd just move it into `_process_POST`.. the main benefit of `_process_args` is to have something that runs for all methods and that can be used easily w/ inheritance. But that's nothing we need here..